django/views/generic/edit.py

```diff
@@ -163,6 +163,7 @@ def get_success_url(self):
         Returns the supplied URL.
         """
         if self.success_url:
+            self.success_url = force_text(self.success_url)
```

Comment:
Hmm, that's a good idea. However, I can't find tests with new format-style placeholders. Does `obj.format` works when `obj` is lazy object?